src/pudl/output/eia860.py

```diff
@@ -317,10 +334,14 @@ def generators_eia860(pudl_engine, start_date=None, end_date=None):
         .astype({
             "plant_id_eia": "Int64",
             "plant_id_pudl": "Int64",
+            "unit_id_pudl": "Int64",
```

Comment:
Rather than try to chase down all these instances, I think we should create an issue and integrate it better post-harvesting. There are also better data type enforcement tools getting built into pandas now.